v1/pipeline/steps/cd/k8s-apply-step-info.yaml

```diff
@@ -3,99 +3,45 @@ allOf:
 - $ref: ../../common/step-spec-type.yaml
 - oneOf:
   - required:
-    - filePaths
+      - paths
   - required:
-    - manifestSource
+      - manifest_source
 - type: object
   properties:
-    commandFlags:
+    flags:
+      description: command_flags
       oneOf:
-      - type: array
-        items:
-          $ref: k8s-step-command-flag.yaml
-      - type: string
-        pattern: ^<\+input>((\.)((executionInput\(\))|(allowedValues|default|regex)\(.+?\)))*$
-        minLength: 1
-    delegateSelectors:
+        - type: array
+          items:
+            $ref: k8s-step-command-flag.yaml
+        - type: string
+    delegates:
+      $ref: ../../common/delegate.yaml
+    paths:
       oneOf:
-      - type: array
-        items:
-          type: string
-      - type: string
-        pattern: (<\+.+>.*)
-        minLength: 1
-    filePaths:
-      oneOf:
-      - type: array
-        items:
-          type: string
-        maxItems: 256
-        minItems: 1
-      - type: string
-        pattern: ^<\+input>((\.)((executionInput\(\))|(allowedValues|default|regex)\(.+?\)))*$
-        minLength: 1
+        - type: array
+          items:
+            type: string
+          maxItems: 256
+          minItems: 1
+        - type: string
+          pattern: ^<\+input>((\.)((executionInput\(\))|(allowedValues|default|regex)\(.+?\)))*$
+          minLength: 1
     overrides:
       type: array
       items:
         $ref: manifest-config-wrapper.yaml
```

Comment:
@shalini-agr this `manifest-config-wrapper.yaml` later references the `manifest-config.yaml` and it has V0 schema structure like below. So lets update that as well.
```
required:
  - identifier
  - spec
  - type
 ```